wpiutil/src/main/generated/java/edu/wpi/first/util/datalog/StringLogEntry.java

```diff
@@ -2,37 +2,16 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
+// THIS FILE WAS GENERATED BY generate_datalog_entries.py DO NOT EDIT IT MANUALLY
+
 package edu.wpi.first.util.datalog;
 
-/** Log string values. */
-public class StringLogEntry extends DataLogEntry {
-  /** The data type for string values. */
-  public static final String kDataType = "string";
 
-  /**
-   * Constructs a String log entry.
-   *
-   * @param log datalog
-   * @param name name of the entry
-   * @param metadata metadata
-   * @param type Data type
-   * @param timestamp entry creation timestamp (0=now)
-   */
-  public StringLogEntry(DataLog log, String name, String metadata, String type, long timestamp) {
-    super(log, name, type, metadata, timestamp);
-  }
```

Comment:
We should figure out how to make the constructors match what was there previously.